### DISCUSSION TITLE
Support 3DTiles styling of Cesium

### DISCUSSION THREAD

**Discussion by valentinMachado** [iTowns/itowns#2085] [Ideas]
Cesium propose a styling for the 3DTiles [ref](https://github.com/CesiumGS/3d-tiles/tree/main/specification/Styling)
It could be nice to support it in itowns. Maybe by creating a parser which would convert these styling.json files into itowns.Style objects.

**Comment by alexLuky** [comment]
I think first we need to render in same quality as Cesium does. Here is a [link](https://github.com/iTowns/itowns/discussions/2076) to another discussion about this.
